src/ngSanitize/sanitize.js

```diff
@@ -262,11 +236,34 @@ function makeMap(str, lowercaseKeys) {
   return obj;
 }
 
+var baseNode;
+(function(window) {
+  var doc;
+  if (window.DOMDocument) {
+    doc = new window.DOMDocument();
+  } else if (window.document && window.document.implementation) {
+    doc = window.document.implementation.createHTMLDocument("inert");
+  } else if (window.ActiveXObject) {
+    doc = new ActiveXObject("Msxml.DOMDocument");
+  } else {
+    throw "DOMDocument not supported!";
+  }
+  var docElement = doc.documentElement || doc.getDocumentElement();
+  var bodys = docElement.getElementsByTagName('body');
+  if (bodys.length === 1) {
+    baseNode = bodys[0];
+  } else {
+    var html = doc.createElement('html');
+    var body = baseNode = doc.createElement('body');
+    html.appendChild(body);
+    doc.appendChild(html);
+  }+
+})(window);
```

Comment:
there is no drawback, but I don't think there is any benefit either. we need to test this against the real window.